integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/tpcxbb/TpcxbbLikeBench.scala

```diff
@@ -19,7 +19,7 @@ package com.nvidia.spark.rapids.tests.tpcxbb
 import com.nvidia.spark.rapids.tests.common.BenchUtils
 
 import org.apache.spark.internal.Logging
-import org.apache.spark.sql.{DataFrame, SparkSession}
+import org.apache.spark.sql.{DataFrame, SaveMode, SparkSession}
```

Comment:
Not sure `SafeMode` is utilized anymore?

Reply:
Fixed